homeassistant/components/tuya/sensor.py

```diff
@@ -594,6 +594,24 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             subkey="voltage",
         ),
     ),
+	# Robot Vacuum
+    # https://developer.tuya.com/en/docs/iot/fsd?id=K9gf487ck1tlo
```

Comment:
Should we add support for the others too? Like `total_clean_area`, `total_clean_count`, etc?